### PR TITLE
community/imapsync: move to community & update to 1.727

### PR DESCRIPTION
**Pull request by itoffshore**
the `imapsync` install script runs `perl -m$MODULE::NAME` on the dependent modules so I am adding the new perl modules straight into community as their dependencies are already checked:

```
sh INSTALL.d/prerequisites_imapsync
$SHELL says  /bin/ash
$0 gives  INSTALL.d/prerequisites_imapsync
ps -ef gives 116661root       0:00 sh INSTALL.d/prerequisites_imapsync
Linux musl64 4.4.20-1-MANJARO #1 SMP PREEMPT Wed Sep 7 19:11:34 UTC 2016 x86_64 Linux
Ok: Found Perl 5.24.0
Ok: Found Perl module Authen::NTLM
Ok: Found Perl module Compress::Zlib
Ok: Found Perl module Data::Dumper
Ok: Found Perl module Data::Uniqid
Ok: Found Perl module Digest::HMAC_MD5
Ok: Found Perl module Digest::HMAC
Ok: Found Perl module Digest::MD5
Ok: Found Perl module File::Copy::Recursive
Ok: Found Perl module IO::Socket::INET
Ok: Found Perl module IO::Socket::INET6
Ok: Found Perl module IO::Socket::SSL
Ok: Found Perl module IO::Tee
Ok: Found Perl module JSON::WebToken
Ok: Found Perl module Mail::IMAPClient
Ok: Found Perl module Parse::RecDescent
Ok: Found Perl module Readonly
Ok: Found Perl module Term::ReadKey
Ok: Found Perl module Test::MockObject
Ok: Found Perl module Test::More
Ok: Found Perl module Test::Pod
Ok: Found Perl module Unicode::String
Ok: Found Perl module URI::Escape
All needed modules are already installed
imapsync syntax OK
```

### CODE REVIEW

**Comment by jirutka** [comment]
The perl abuilds are again full of indentation spaces…

**Comment by itoffshore** [comment]
sorry - fixed now

**Comment by itoffshore** [comment]
sorry - fixed now

File: community/imapsync/APKBUILD [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-pkgver=1.678
-pkgrel=1
+pkgver=1.727
+pkgrel=0
 pkgdesc="IMAP synchronisation, sync, copy or migration tool"
 url="http://imapsync.lamiral.info"
 arch="noarch"
 license="custom"
 depends="perl-digest-hmac perl-authen-ntlm perl-io-compress perl-data-dumper \
 	perl-data-uniqid perl-digest-md5 perl-file-copy-recursive perl-io-socket-inet6 \
```
Comment (jirutka) [inline community/imapsync/APKBUILD]: `perl-io-socket-inet6` is in testing, so it must be moved to the community first.

**Comment by itoffshore** [comment]
I moved `perl-io-socket-inet6` - but the rebuild failed due to:

`ERROR: perl-mime-base64-3.15-r2: temporary error (try again later)`

the package seems to be available - can you trigger another rebuild or do I need to alter something in the `APKBUILD` to trigger it ?

**Pull request closed without merging**